Flat/interface/TemplatedPandaAnalyzer.h

```diff
@@ -252,7 +275,13 @@ void PandaAnalyzer::FillGenTree(panda::Collection<T>& genParticles)
     pandaecf::calcECFN(beta+1, sdConstsFiltered, ecfnMan);
     for (int N = 1; N != 5 ; ++N) {
       for (int o = 1; o != 4; ++o) {
-        genJetInfo.ecfs[o-1][N-1][beta-1] = ecfnMan->ecfns[Form("%i_%i", N, o)];
+        float theEcf = ecfnMan->ecfns[Form("%i_%i", N, o)];
+        genJetInfo.ecfs[o-1][N-1][beta-1] = theEcf; 
```

Comment:
You may notice (on line 274) that I only consider beta=1,2 here. This might not be sufficient for your needs.